placements/tests.py

```diff
@@ -49,6 +49,26 @@ def test_training_get(self):
         """Check auth before getting training blog."""
         test_blog(self, '/api/training-blog', 3)
 
+    # Adding test for pin_unpin feature
+
+    def test_blog_order(obj):     # pylint: disable=E0213
```

Comment:
I think class methods should have `self` arg. having some other argument can cause confusion (that is what E213 pep is for). 
You should change this to self, and update all `obj` entries in this test function to `self`, makes intent more clear. (and remove disabled pylint comments)